views/tools/tools.hbs

```diff
@@ -10,6 +10,19 @@
 
     {{#content "page"}}
         <div class="container-fluid">
+            <div class="row mobile-no-margin-right">
+                <div class="card h-100">
+                    <div class="card-block">
+                        <h5 class="card-title">OAuth2 Endpunkte</h5>
+                        <div class="card-text">
+                            <h6>Authorization: <a href="{{baseUrl}}/oauth2/auth">{{baseUrl}}/oauth2/auth</a></h6>
+                            <h6>Token: <a href="{{baseUrl}}/oauth2/token/">{{baseUrl}}/oauth2/token</a></h6>
+                            <h6>Userinfo: <a href="{{baseUrl}}/oauth2/token">{{baseUrl}}/userinfo</a></h6>
```

Comment:
href is wrong here

Reply:
is correct when configured in server config (services.hydra)